cascade/dynamics.py

```diff
@@ -1,136 +0,0 @@
-    timesteps: int | None = None
-    """Maximum number of timesteps to run.
-
-    Use ``None`` to run until :attr:`driver` reports the dynamics as converged"""
-    driver_kwargs: dict[str, Any] = field(default_factory=dict)
-    """Keyword arguments used to create the driver"""
-    run_kwargs: dict[str, Any] = field(default_factory=dict)
-    """Keyword arguments passed to the driver's run method"""
-    post_fun: Optional[Callable[[Atoms], None]] = None
-    """Post-processing function applied after run is complete. Modifies the input arguments"""
-
-
-class DynamicsProtocol:
-    """A protocol for running several stages of dynamics calls together
-
-    Args:
-        stages: List of dynamics to be run in sequential order
-        scratch_dir: Directory in which to write temporary files
-    """
-
-    stages: list[DynamicsStage]
-    """List of dynamics processes to run sequentially"""
-
-    def __init__(self, stages: list[DynamicsStage], scratch_dir: Path | None = None):
-        self.stages = stages.copy()
-        self.scratch_dir = scratch_dir
-
-    # TODO (wardlt): We might need to run dynamics with a physics code, which will require changing the interface
-    def run_dynamics(self,
-                     start: Progress,
-                     model_msg: bytes | State,
-                     learner: BaseLearnableForcefield,
-                     max_timesteps: int,
-                     max_frames: int | None = None,
-                     device: str = None) -> tuple[bool, list[Atoms]]:
-        """Run dynamics for a maximum number of timesteps using a particular forcefield
-
-        Runs dynamics until the end of a process or until the maximum number of timesteps is reached.
-
-        Args:
-            start: Starting point of the dynamic trajectory
-            model_msg: Serialized form of the forcefield used for training
-            learner: Class used to generate the ASE calculator object
-            max_timesteps: Maximum number of timesteps to run dynamics
-            max_frames: Maximum number of frames from the atomistic simulation to return for auditing
-            device: Device to use for evaluating forcefield
-        Returns:
-            List of frames selected at used for auditing the dynamics
-        """
-
-        # Create a temporary directory in which to run the data
-        stage = self.stages[start.stage]  # Pick the current process
-        with TemporaryDirectory(dir=self.scratch_dir, prefix='cascade-dyn_', suffix=f'_{start.name}') as tmp:
-            tmp = Path(tmp)
-            dyn = stage.driver(start.atoms, logfile=str(tmp / 'dyn.log'), **stage.driver_kwargs)
-
-            # Attach the calculator
-            calc = learner.make_calculator(model_msg, device)
-            atoms = start.atoms
-            atoms.calc = calc
-
-            # Attach the trajectory writer
-            traj_freq = 1 if max_frames is None else max_timesteps // max_frames
```

Comment:
Good idea to kill off this stuff. Far more cmplicated than we needed

Reply:
I just figured we had moved on to a new paradigm and there wasnt in here we needed